pkg/operation/botanist/component/vpnshoot/vpnshoot_test.go

```diff
@@ -687,12 +689,16 @@ status:
 					deployment := &appsv1.Deployment{}
 					Expect(runtime.DecodeInto(newCodec(), managedResourceSecret.Data["deployment__kube-system__vpn-shoot.yaml"], deployment)).To(Succeed())
 					Expect(deployment).To(Equal(deploymentFor(secretNameCA, secretNameClient, secretNameTLSAuth, values.ReversedVPN.Enabled, values.VPAEnabled)))
+					Expect(string(managedResourceSecret.Data["rolebinding__kube-system__gardener.cloud_psp_vpn-shoot.yaml"])).To(Equal(roleBindingPSPYAML))
+					Expect(string(managedResourceSecret.Data["podsecuritypolicy____gardener.kube-system.vpn-shoot.yaml"])).To(Equal(podSecurityPolicyYAML))
+					Expect(string(managedResourceSecret.Data["clusterrole____gardener.cloud_psp_kube-system_vpn-shoot.yaml"])).To(Equal(clusterRolePSPYAML))
 				})
 			})
 
-			Context("w/ VPA", func() {
+			Context("w/ VPA, PSP is disabled", func() {
```

Comment:
Similar to one of the previous comments - I guess the VPA is not related to the PSP. Is it possible to extract the PSP testing to 2 new contexts - `w/ PSP disabled` and `w/ PSP not disabled`? This comment also applies for some of the other component tests (like `nodelocaldns`) - to avoid spam we won't comment the same there.